adapters/http-base/src/main/java/org/eclipse/hono/adapter/http/impl/VertxBasedHttpProtocolAdapter.java

```diff
@@ -98,6 +98,8 @@ public String getTypeName() {
 
     @Override
     protected void addRoutes(final Router router) {
+        // ensure route order check is disabled globally to allow auth handler to be added before body handler
+        System.setProperty(HttpUtils.SYSTEM_PROPERTY_ROUTER_SETUP_LENIENT, "true");
```

Comment:
is there a particular reason you want to set this for each HTTP based adapter separately instead of setting it in the superclass?

Reply:
It was about defining it in the same method where the handlers get added.
But I'm also fine with adding it in the superclass. I've amended the commit accordingly.